logger/common.go

```diff
@@ -64,6 +62,8 @@ type GlobalArgs struct {
 	UID           int
 	GID           int
 	CleanupTime   *time.Duration
+	ProxyEnvVar   string
```

Comment:
Can you extract these two fields to a separate struct and then add the new struct there?

Reply:
Do we plan to only support proxy environment variable settings on Windows?